eos/effects/modulebonusnetworkedsensorarray.py

```diff
@@ -4,13 +4,18 @@
 # Module: Networked Sensor Array
 type = "active"
 def handler(fit, src, context):
-    fit.ship.multiplyItemAttr("maxTargetRange", src.getModifiedItemAttr("maxTargetRangeMultiplier"), stackingPenalties=True, penaltyGroup="postMul")
     fit.ship.boostItemAttr("scanResolution", src.getModifiedItemAttr("scanResolutionBonus"), stackingPenalties=True)
 
     for scanType in ('Magnetometric', 'Ladar', 'Gravimetric', 'Radar'):
         fit.ship.boostItemAttr("scan{}Strength".format(scanType),
                                src.getModifiedItemAttr("scan{}StrengthPercent".format(scanType)),
                                stackingPenalties=True)
+                               
+    for fighter in fit.fighters:
+        for scanType in ('Magnetometric', 'Ladar', 'Gravimetric', 'Radar'):
+            fighter.boostItemAttr("scan{}Strength".format(scanType),
+                               src.getModifiedItemAttr("scan{}StrengthPercent".format(scanType)),
+                               stackingPenalties=True)
```

Comment:
Instead of looping over the fighters themselves, try something like this:

```
for scanType in ('Magnetometric', 'Ladar', 'Gravimetric', 'Radar'):         
    fit.fighters.filteredItemBoost(lambda mod: mod.item.requiresSkill("Fighters"), "scan{}Strength".format(scanType), getModifiedItemAttr("scan{}StrengthPercent".format(scanType)), True)
```

(untested)

This basically accomplishes the same thing, but it's the construct uses in nearly all other effect files.